docs/reference/limitations/openstack.md

```diff
@@ -20,6 +20,15 @@ It also means that you cannot use the following `openstack` CLI commands; they a
   > You can use `openstack project list`, but this will only list the project(s) that your user account has access to.
 * `openstack domain <subcommand>`
 
+## Cinder
+
+### Volume backup service
+
+The volume backup service is not configured in any of the {{brand}} regions.
+As a result, volume backups cannot be created.
```

Comment:
```suggestion
{{brand}} does not support the OpenStack volume backup service (`cinder-backup`).

For automated, scheduled volume *snapshots,* consider configuring your servers for [Disaster Recovery (DR)](../../background/disaster-recovery.md) via the {{gui}}.
```

Reply:
It is ever so slightly better than my wording, so let's go ahead with yours :)